pkg/rancher-desktop/assets/dependencies.yaml

```diff
@@ -2,15 +2,15 @@ limaAndQemu: 1.31.2
 alpineLimaISO:
   isoVersion: 0.2.30.rd4
   alpineVersion: 3.18.0
-WSLDistro: "0.41"
+WSLDistro: "0.42"
 kuberlr: 0.4.2
-helm: 3.12.0
+helm: 3.12.1
```

Comment:
Sorry, I screwed up the bump, recreated as #5045.

Reply:
Thanks for noticing! The cherry-pick is still good though, isn't it? It just comes from the branch, but will be identical to the one from the new PR.

Reply:
Yeah, the cherry-pick is fine (which is why I approved it), I just got confused comparing it to the copy on `main`.